### ISSUE TITLE
Use single editor.json resource for autocompletion list

### ISSUE THREAD

**Issue by rowanseymour** [nyaruka/temba-components#48]
Mailroom now shares /mr/docs/editor.json which looks like:

```
{
  "context": { ... what used to be in completion.json ..   }
  "functions": { ... what used to be in functions.json ..   }
}
```

**Comment by rowanseymour** [comment]
To be clear it also still shares the current files so nothing breaks for now.

**Comment by ericnewcomer** [comment]
Fixed by #82
